### DISCUSSION TITLE
Letter-by-letter pronunciation in XTTS

### DISCUSSION THREAD

**Discussion by madey83** [idiap/coqui-ai-TTS#196] [Q&A] [answered]
is there any way to force model to do not generate voice for shortcut like "PGNCS" instead just to do literalize individual letters

**Comment by eginhard** [comment, marked as answer]
You can try putting spaces or period between letters, but as already explained in #195 you can't force a specific pronunciation with XTTS.

**Comment by madey83** [comment]
Can you tell which model can do that? I'm asking beacues i heard voice which can prenauce CIA, "Silos" correctly in polish langueage...

**Comment by madey83** [comment]
n/a
